app/views/activity_forms/objectives.html.haml

```diff
@@ -1,5 +1,4 @@
 = render layout: "wrapper" do |f|
-  = f.govuk_fieldset legend: { text: t("form.legend.activity.objectives", level: t("page_content.activity.level.#{f.object.level}")), tag: :h1, size: "xl" } do
```

Comment:
This is intentional. The fieldset shouldn't be necessary when there's only one input. The label just said "Aims/Objectives of the activity", which adds nothing to the question "What are the aims/objectives of this %{level}?", so the label of the remaining text area is now the full question and the original label is removed